src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -170,7 +172,31 @@
       <TfmSpecificPackageFile Include="$(OutputPath)\$(ILMergeSubpath)NuGet.Build.Tasks.Pack.dll">
         <PackagePath>CoreCLR/</PackagePath>
       </TfmSpecificPackageFile>
-      <TfmSpecificPackageFile Include="$(OutputPath)\$(ILMergeSubpath)NuGet.Build.Tasks.Pack.xml">
+      <TfmSpecificPackageFile Include="$(OutputPath)\$(ILMergeSubpath)**\NuGet*.resources.dll">
+        <PackagePath>CoreCLR/</PackagePath>
+      </TfmSpecificPackageFile>
+      <TfmSpecificPackageFile Include="$(OutputPath)\$(ILMergeSubpath)**\*.dll" Condition="'$(IsBuildOnlyXPLATProjects)' == 'true'">
+        <PackagePath>CoreCLR/</PackagePath>
+      </TfmSpecificPackageFile>
+    </ItemGroup>    
+  </Target>
+
+   <!--These targets help get files for packing the second NuGet.Build.Tasks.Pack package, for netstandard2.1.-->
```

Comment:
This package is not the netstandard2.1 one. It's the netstandard2.0 one.

Reply:
Thanks! Corrected the wrong naming by exchanging them.